app/assets/stylesheets/forms/_inputs.scss

```diff
@@ -155,35 +155,35 @@ input[type=checkbox] {
 	&:checked::after {
 		content: "";
 		display: inline-block;
-		border-left: 2px solid;
-		border-bottom: 2px solid;
+		border-left: .125rem solid;
+		border-bottom: .125rem solid;
 		transform: rotate(-45deg);
 		position: absolute;
 
-		height: 5px;
-	    top: 2px;
-	    width: 11px;
-	    left: 1px;
+		height: .313rem;
+	    top: .125rem;
+	    width: .688rem;
+	    left: .062rem;
 	}
 }
 
 input[type=range] {
 	-webkit-appearance: none;
 	border: none;
 	background: transparent;
-	margin: 25px 0px 20px;
+	margin: 1.563rem 0rem 1.25rem;
```

Comment:
i always follow the "if ur gonna write 3, write 4" rule when it comes to trbl.

`margin: 25px 0px 20px 0px` is way more legible than `margin: 25px 0px 20px`, and so is:
`margin: 25px 0`
`margin-bottom: 20px`

what are you really saving by that other than keystrokes? worth it? eh